src/ProgramBlockEditor.js

```diff
@@ -189,6 +205,7 @@ class ProgramBlockEditor extends React.Component<ProgramBlockEditorProps, {}> {
                     </Col>
                     <div className='ProgramBlockEditor__editor-actions'>
                         <Button
+                            disabled={this.props.activeProgramStepNum !== null}
```

Comment:
I'd like to make this logic more explicit: let's add a new prop to `ProgramBlockEditor` called `editingDisabled` and put the logic for disabling (`interpreterIsRunning === true`) in `App.js`